source/extensions/filters/http/cache/cache_filter.h

```diff
@@ -46,16 +46,30 @@ enum class FilterState {
   Destroyed
 };
 
+class CacheFilterConfig {
```

Comment:
This would be simpler as a struct with public members, since it has no invariants

Reply:
It's probably going to end up with other things later, and also this way is more similar to the common filter config format.

Reply:
Specific example, I have a PR in the works that's going to add a shared pointer to a thundering herd manager structure in here.

Reply:
(Which probably could still just be a public struct member, and if I was the style maven here I would absolutely do that, but I'm trying to be conformant with how everything else is, and classes with accessors seems to be the standard for preprocessed filter configs.)

Reply:
"class" sounds good. (I looked at envoy classes named "*Config", and the ratio of "class" to "struct" is about 10:1, and consistency is a useful property.)